[Bug]: PublicSite builds are failing

### What kind of bug is this?

Other (Provide below)

### If you selected "Other" in the previous section, what kind of bug is it?

CI/CD

### Describe the bug.

PublicSite builds are failing due to a change from .NET 8 RC1 -> RC2. The fix is to update the attributes/methods to their new names.

### Logs (Optional)

```plain text
2023-10-11T14:26:44.9990335Z MSBuild version 17.8.0+6cdef4241 for .NET
2023-10-11T14:26:45.8016424Z   Determining projects to restore...
2023-10-11T14:26:46.7383568Z   All projects are up-to-date for restore.
2023-10-11T14:26:51.6337011Z   Lib -> /home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/Lib/bin/Debug/net8.0/SmallsOnline.Web.Lib.dll
2023-10-11T14:26:54.5001722Z   Lib.Services -> /home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/Lib.Services/bin/Debug/net8.0/SmallsOnline.Web.Lib.Services.dll
2023-10-11T14:26:55.2494698Z   Lib.Components -> /home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/Lib.Components/bin/Debug/net8.0/SmallsOnline.Web.Lib.Components.dll
2023-10-11T14:26:57.6314597Z ##[error]/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Shared/Sidebar/SidebarItem.razor(12,170): error CS0246: The type or namespace name 'ServerRenderMode' could not be found (are you missing a using directive or an assembly reference?) [/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Server.csproj]
2023-10-11T14:26:57.6325304Z ##[error]/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Program.cs(13,6): error CS1061: 'IRazorComponentsBuilder' does not contain a definition for 'AddServerComponents' and no accessible extension method 'AddServerComponents' accepting a first argument of type 'IRazorComponentsBuilder' could be found (are you missing a using directive or an assembly reference?) [/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Server.csproj]
2023-10-11T14:26:57.6328626Z ##[error]/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Program.cs(41,6): error CS1061: 'RazorComponentsEndpointConventionBuilder' does not contain a definition for 'AddServerRenderMode' and no accessible extension method 'AddServerRenderMode' accepting a first argument of type 'RazorComponentsEndpointConventionBuilder' could be found (are you missing a using directive or an assembly reference?) [/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Server.csproj]
2023-10-11T14:26:57.6395672Z 
2023-10-11T14:26:57.6395836Z Build FAILED.
2023-10-11T14:26:57.6395987Z 
2023-10-11T14:26:57.6397903Z ##[error]/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Shared/Sidebar/SidebarItem.razor(12,170): error CS0246: The type or namespace name 'ServerRenderMode' could not be found (are you missing a using directive or an assembly reference?) [/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Server.csproj]
2023-10-11T14:26:57.6401367Z ##[error]/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Program.cs(13,6): error CS1061: 'IRazorComponentsBuilder' does not contain a definition for 'AddServerComponents' and no accessible extension method 'AddServerComponents' accepting a first argument of type 'IRazorComponentsBuilder' could be found (are you missing a using directive or an assembly reference?) [/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Server.csproj]
2023-10-11T14:26:57.6404566Z ##[error]/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Program.cs(41,6): error CS1061: 'RazorComponentsEndpointConventionBuilder' does not contain a definition for 'AddServerRenderMode' and no accessible extension method 'AddServerRenderMode' accepting a first argument of type 'RazorComponentsEndpointConventionBuilder' could be found (are you missing a using directive or an assembly reference?) [/home/runner/work/SmallsOnline.Web/SmallsOnline.Web/src/PublicSite/Server/Server.csproj]
2023-10-11T14:26:57.6405796Z     0 Warning(s)
2023-10-11T14:26:57.6405987Z     3 Error(s)
2023-10-11T14:26:57.6406108Z 
2023-10-11T14:26:57.6406230Z Time Elapsed 00:00:12.51
2023-10-11T14:26:58.5915156Z ##[error]Process completed with exit code 1.
```

--- a/src/PublicSite/Client/Components/Sidebar/SidebarProfileImgWasm.razor.cs
+++ b/src/PublicSite/Client/Components/Sidebar/SidebarProfileImgWasm.razor.cs
@@ -3,6 +3,6 @@ using Microsoft.AspNetCore.Components.Web;
 
 namespace SmallsOnline.Web.PublicSite.Client.Components.Sidebar;
 
-[RenderModeAuto(true)]
+[RenderModeInteractiveAuto(true)]
 public partial class SidebarProfileImgWasm : SidebarProfileImg
 {}

--- a/src/PublicSite/Server/Program.cs
+++ b/src/PublicSite/Server/Program.cs
@@ -10,7 +10,7 @@ builder.Configuration
 // Add services to the container.
 builder.Services
     .AddRazorComponents()
-    .AddServerComponents();
+    .AddInteractiveServerComponents();
 
 builder.Services
     .AddRazorPages();
@@ -38,7 +38,7 @@ app.UseStaticFiles();
 
 app
     .MapRazorComponents<App>()
-    .AddServerRenderMode();
+    .AddInteractiveServerRenderMode();
 
 app.MapRazorPages();
 //app.MapFallbackToPage("/error");

--- a/src/PublicSite/Server/Shared/Sidebar/SidebarItem.razor
+++ b/src/PublicSite/Server/Shared/Sidebar/SidebarItem.razor
@@ -9,7 +9,7 @@
         <div class="card-body">
             <div class="d-flex justify-content-center">
                 <div class="flex-shrink-1">
-                    <SidebarProfileImg LoggingEnabled="@_isDevelopmentMode" @rendermode="@(new ServerRenderMode(true))" />
+                    <SidebarProfileImg LoggingEnabled="@_isDevelopmentMode" @rendermode="@(new InteractiveServerRenderMode(true))" />
                 </div>
             </div>
             <div class="row justify-content-center gx-0 mt-4">

